tests/hash-backend-integration/src/tests/email.test.ts

```diff
@@ -3,6 +3,7 @@ import "./load-test-env";
 import { AwsSesEmailTransporter } from "@apps/hash-api/src/email/transporters";
 import { getAwsRegion } from "@apps/hash-api/src/lib/aws-config";
 import { getRequiredEnv } from "@local/hash-backend-utils/environment";
+import { it } from "vitest";
 
 if (process.env.HASH_DEV_INTEGRATION_EMAIL) {
   const to = process.env.HASH_DEV_INTEGRATION_EMAIL;
```

Comment:
We have this block down below, maybe we can get rid of it now? Or remove reference to 'jest'
```ts
} else {
  // Need at least one test otherwise jest will complain
  it("always passes", () => undefined);
}
```

Reply:
`vitest` still needs at least a single test in a file, but there's a proper way to skip a test instead.

see: [ffd08ac](https://github.com/hashintel/hash/pull/4180/commits/ffd08ac15679fc16920132071c6506803ae577d4)